docs/core/porting/libraries.md

```diff
@@ -13,336 +13,183 @@ ms.assetid: a0fd860d-d6b6-4659-b325-8a6e6f5fa4a1
+## .NET Framework technologies unavailable on .NET Core
 
-It's recommended that you pick the lowest possible .NET Standard version and use that throughout your project.
+Several technologies available to .NET Framework libraries aren't available for use with .NET Core, such as AppDomains, Remoting, Code Access Security (CAS), and Security Transparency. If your libraries rely on one or more of these technologies, consider the alternative approaches outlined below. For more information on API compatibility, the CoreFX team maintains a [List of behavioral changes/compat breaks and deprecated/legacy APIs](https://github.com/dotnet/corefx/wiki/ApiCompat) at GitHub.
 
-Read more in [.NET Platform Standard Library](../../standard/net-standard.md).
+Just because an API or technology isn't currently implemented doesn't imply it's intentionally unsupported. Feel free to file an issue in the [dotnet/corefx repository issues](https://github.com/dotnet/corefx/issues) at GitHub to ask for specific APIs and technologies. [Porting requests in the issues](https://github.com/dotnet/corefx/labels/port-to-core) are marked with the `port-to-core` label.
 
-## Key Technologies Not Yet Available on the .NET Standard or .NET Core
+### AppDomains
 
-You may be using some technologies available for the .NET Framework that are not currently available for .NET Core. Each of the following sub-sections corresponds to one of those technologies. Alternative options are listed if it is feasible for you to adopt them.
+AppDomains isolate apps from one another. AppDomains require runtime support and are generally quite expensive. They're not implemented in .NET Core. We don't plan on adding this capability in future.
 
-### App Domains
+For code isolation, we recommend separate processes or using containers as an alternative. For the dynamic loading of assemblies, we recommend the new <xref:System.Runtime.Loader.AssemblyLoadContext> class. Information (such as the name and base directory) is provided by APIs on other types, for instance <xref:AppContext.BaseDirectory%2A>. Some scenarios, such as getting the list of loaded assemblies, are unsupported, as they're inherently fragile.
 
-AppDomains can be used for different purposes on the .NET Framework. For code isolation, we recommend separate processes and/or containers as an alternative. For dynamic loading of assemblies, we recommend the new @System.Runtime.Loader.AssemblyLoadContext class.
+To make code migration from .NET Framework easier, we've exposed some of the <xref:System.AppDomain> API surface in .NET Core. Some of the APIs function normally (for example, <xref:System.AppDomain.UnhandledException?displayProperty=fullName>), some of them do nothing (for example, <xref:System.AppDomain.SetCachePath%2A>), and some of them throw <xref:System.PlatformNotSupportedException> (for example, <xref:System.AppDomain.CreateDomain%2A>).
```

Comment:
You should add a sentence about where to find which classes and APIs take which action (even if it is the obvious "check the types you use for details".)